pycytominer/cyto_utils/cell_locations.py

```diff
@@ -147,7 +147,7 @@ def _s3_file_exists(self, s3_path: str):
         bucket, key = self._parse_s3_path(s3_path)
 
         try:
-            self.s3.Object(bucket, key).load()
+            self.s3.head_object(Bucket=bucket, Key=key)
         except botocore.exceptions.ClientError as e:
             if e.response["Error"]["Code"] == "404":
```

Comment:
While not a change for this PR, would we also want to catch `400`, or `403` errors somehow (recognizing they may result in similar circumstances)?

Reply:
That makes sense; addressed